woocommerce/woocommerce
# Added workflow for running smoke tests on PR

### Changes proposed in this Pull Request:

This PR adds a new workflow to run smoke tests against an external host based on the label. This currently uses the daily smoke test host.

Closes https://github.com/woocommerce/woocommerce/issues/30614.

### How to test the changes in this Pull Request:

1. Verify the workflow syntax looks correct, notably in the `if` check.
2. Review the workflow run against this PR.
